gui/wxpython/gui_core/goutput.py

```diff
@@ -416,6 +445,17 @@ def OnCmdProgress(self, event):
         self.progressbar.SetValue(event.value)
         event.Skip()
 
+    def UpdateHistory(self, cmd):
+        """Update command history"""
+        history_path = get_current_mapset_gui_history_path()
+        try:
+            update_history(cmd, history_path)
+            self.showNotification.emit(
+                message=_("Command history saved to '{}'".format(history_path))
```

Comment:
Seems unnecessary to inform user every time you execute a command, I would probably remove this.